Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/StaticOfferDriver/Ride.hs

```diff
@@ -79,8 +84,20 @@ handler merchantId =
     :<|> rideCancel merchantId
     :<|> rideInfo merchantId
     :<|> rideSync merchantId
+    :<|> multipleRideSync merchantId
     :<|> rideRoute merchantId
 
+buildTransactionMultipleRide ::
+  ( MonadFlow m,
+    Common.HideSecrets request
+  ) =>
+  Common.RideEndpoint ->
+  ApiTokenInfo ->
+  Maybe request ->
+  m DT.Transaction
+buildTransactionMultipleRide endpoint apiTokenInfo =
+  T.buildTransaction (DT.RideAPI endpoint) (Just BECKN_TRANSPORT) (Just apiTokenInfo) Nothing Nothing
```

Comment:
same as above these two can also be merged.

Reply:
fixed it